dev/com.ibm.ws.microprofile.rest.client_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -0,0 +1,256 @@
+            <artifactId>arquillian-wlp-managed-8.5</artifactId>
+            <!-- <version>1.0.0.Final-SNAPSHOT</version> -->
+            <version>1.0.0.Final-20180108.155248-1</version>
+            <scope>test</scope>
+        </dependency>
+
+<!-- 
+        <dependency>
+            <groupId>com.github.tomakehurst</groupId>
+            <artifactId>wiremock</artifactId>
+            <version>2.15.0</version>
+            <type>pom</type>
+            <scope>test</scope>
+        </dependency>
+ --> 
+ 
+        <dependency>
+            <groupId>com.github.tomakehurst</groupId>
+            <artifactId>wiremock-standalone</artifactId>
+            
+            <version>2.14.0</version>
+            <!-- <scope>system</scope>
+            <systemPath>${basedir}/../../servers/FATServer/wiremock-standalone-2.14.0.jar</systemPath> -->
+        </dependency>
+        
+        
+    </dependencies>
+ 
+    <build>
+        <directory>${targetDirectory}</directory>
+        <defaultGoal>clean test</defaultGoal>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <version>2.9</version>
+                <executions>
+                    <execution>
+                        <id>build-classpath</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>build-classpath</goal>
+                        </goals>
+                        <configuration>
+                            <!-- configure the plugin here -->
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>2.3.2</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>2.17</version>
+                <configuration>
+                    <!-- <argLine>-Dwas.debug.mode=true -Dcom.ibm.websphere.ras.inject.at.transform=true -Dsun.reflect.noInflation=true -agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=7777</argLine> -->
+                    <systemPropertyVariables>
+                        <wlp>${wlp}</wlp>
+                        <!-- <wlp>/home/andymc/mpRestClientTCK/open-liberty/dev/build.image/wlp</wlp> -->
+                        <!-- <tck_server>${tck_server}</tck_server> -->
+                        <tck_server>FATServer</tck_server>
+                        <!-- <tck_port>${tck_port}</tck_port>  -->
```

Comment:
This property is set automatically from the simplicity LibertyServer object that runs the tests in MvnUtils.init. Enabling it to be controlled in the 'traditional way' for a Liberty FAT.